app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+   before_action :configure_sign_up_params, only: [:create]
+  # before_action :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+   def new
+     super
```

Comment:
Is there any reason to define `#new` here? If this methods just calls `super`, then it's the same as just using the inherited method from `Devise::RegistrationsController`. 🤔